Cargo.toml

```diff
@@ -18,44 +18,50 @@ build = "build.rs"
 
 [dependencies]
 base64          = "^0.13"
+basic-cookies   = { version = "^0.1", optional = true }
 bcder           = "0.5.0"
 bytes           = "^0.5"
 chrono          = { version = "^0.4", features = ["serde"] }
 clap            = "^2.33"
 clokwerk        = "^0.3"
-derive_more     = "^0.13"
 fern            = { version = "^0.5", features = ["syslog-4"] }
 futures         = "^0.3"
 futures-util    = "^0.3"
 hex             = "^0.4"
 hyper           = "^0.13"
 intervaltree    = "0.2.6"
-lazy_static     = "1.4.0"
+jmespatch       = { version = "^0.3", features = ["sync"], optional = true }
 libflate        = "^1.0"
 log             = "^0.4"
+openidconnect   = { version = "^1.0", optional = true }
+#openidconnect = { git = "https://github.com/ximon18/openidconnect-rs.git", branch = "krill-hack", optional = true }
 openssl         = { version = "^0.10", features = ["v110"] }
+oso             = { version = "^0.8", optional = true }
```

Comment:
Latest is now 0.9.0 but I'm not aware of any particular bug fixes or features that would be an urgent reason to upgrade.